src/storage/recovery/recovery_manager.cpp

```diff
@@ -320,7 +320,7 @@ void RecoveryManager::UpdateIndexesOnTable(transaction::TransactionContext *txn,
       if (table_pr->IsNull(pr_map[table_col_oid])) {
         index_pr->SetNull(index->GetKeyOidToOffsetMap().at(index_col_oid));
       } else {
-        auto size = col.AttrSize() & INT8_MAX;
+        auto size = col.AttrSize() & INT_MAX;
```

Comment:
INT_MAX might be misinterpreted. The initial way I would interpret this is that this is a 32 bit int max since that's the default size of int. But in this case I think we are looking for a 16 bit int max.

Reply:
This looks like I slipped up. Nice catch!